website/content/docs/auth/azure.mdx

```diff
@@ -390,3 +395,74 @@ namespace Examples
 </CodeBlockConfig>
 
 </CodeTabs>
+  
+## Authentication Flows and Role Restrictions
+  
+This section provides more details about the authentication flows and associated restrictions that can be supplied to the the role definition.
+  
+There are 3 core flows and associated methods of restricting a role in the Azure Auth Method:
+- JWT Flow: Identity-based restrictions that use claims from the JWT.
+- VM Flow: Virtual Machine (VM)-based restrictions that use data from the virtual machine.
+- VMSS Flow: Virtual Machine Scale Set (VMSS)-based restrictions that use data from the Virtual Machine Scale Set.
+  
+The VM and VMSS-based restrictions can only be used for those resource types. If using Azure Kubernetes Service, Azure Container Instance, or some other resource type that supports a managed identity, you cannot apply the restrictions applicable to VM and VMSS.
+  
+The restrictions that apply only to VM or VMSS resource types are:
+- Locations `bound_locations`.
+- Subscriptions `bound_subscription_ids`.
+- Resource groups `bound_resource_groups`.
+- Scale sets `bound_scale_sets`. (VMSS only).
+  
+The restrictions that apply to any resource type are sourced from the JWT, these are:
+- Service principals `bound_service_principal_ids`.
+- Azure AD groups `bound_group_ids`.
+  
+If any of the restrictions for VM or VMSS are applied, then the `vm_name` or `vmss_name` parameters of the [login method](/api-docs/auth/azure#login) become mandatory. These restrictions are optional for VM and VMSS; it is not mandatory to apply any of them.
```

Comment:
I also see `vm_name` being provided when `jwt` is provided on login in [this example](https://www.vaultproject.io/docs/auth/azure#via-the-cli), does this become mandatory for `jwt` as well if any of its restrictions is provided?